consensus/istanbul/core/core.go

```diff
@@ -353,7 +353,7 @@ func (c *core) newRoundChangeTimer() {
 	c.stopTimer()
 
 	// set timeout based on the round number
-	timeout := time.Duration(c.config.RequestTimeout) * time.Millisecond
+	timeout := time.Duration(c.config.Timeout) * time.Millisecond
```

Comment:
Is it okay to use c.config?

Reply:
@kjhman21 Thank you for pointing this issue. Now it is changed to use `DefaultConfig` and confirmed that timeout is changed by the governance.vote.